docs/architecture/cloud-native/PlaceholderForCahcing.md

```diff
@@ -0,0 +1,27 @@
+## Azure Redis Cache
```

Comment:
is this a new article? file name has typos and not following our usual convention,